.github/workflows/main.yml

```diff
@@ -70,3 +71,19 @@ jobs:
     with:
       version: ${{ needs.version.outputs.nextVersionTag }}
       prerelease: ${{ github.event_name == 'pull_request' }}
+  cascading:
```

Comment:
shouldn't we wait for[build, test, lint] to be done before cascading

Reply:
To be discussed, are we waiting for publish or not ?